### PR TITLE
feat: new script to add profile data field

### PR DESCRIPTION
**Pull request by danielgent**
Doing the migration as an initial PR always seems to cause less issues later :smile: 

(I still can't actually develop this though as SSO doesn't work on wrangler)

### CODE REVIEW

File: src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Kysely } from "kysely";
+import { Database } from "../../types";
+
+export async function up(db: Kysely<Database>): Promise<void> {
+  await db.schema
+    .alterTable("users")
+    .addColumn("profileData", "varchar(65535)")
```
Comment (danielgent) [inline src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts:7]: any reason not to use maxlength?

it's not ridiculously long!

Comment (markusahlstrand) [reply src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts:7]: I think planetscale didn't like this. Not sure what the actual max length is there.

**Review by markusahlstrand — APPROVED** [review]
Looks good, but think it's to big for planetscale

**Comment by danielgent** [comment]
> Looks good, but think it's to big for planetscale

Where are you reading this @markusahlstrand ? The planetscale docs say this with no caveat saying planetscale doesn't support this -> https://planetscale.com/blog/mysql-data-types-varchar-and-char#varchar-vs-char

**Comment by markusahlstrand** [comment]
> > Looks good, but think it's to big for planetscale
> 
> Where are you reading this @markusahlstrand ? The planetscale docs say this with no caveat saying planetscale doesn't support this -> https://planetscale.com/blog/mysql-data-types-varchar-and-char#varchar-vs-char

I'm not reading. It's empirical. When I tried to run a similar migration it failed with the error that it was to long :)

File: src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts (hunk 1 above)
Comment (danielgent) [reply src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts:7]: I figure I'll run the migration then and report back? if it doesn't work, I'll update the PR

Oooo, I can do this on a branch actually now and see

Comment (danielgent) [reply src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts:7]: and create the column straight in the planetscale console :relieved:  KISS!

Comment (danielgent) [reply src/migrate/migrations/2023-11-23T17:44:00_profile-data-field.ts:7]: Indeed @markusahlstrand `Column length too big for column 'long_field' (max = 16383);`

image

**Comment by danielgent** [comment]
I'll migrate the db now

**Pull request merged**